apps/storefront/pages/grunnleggende/introduksjon/om-designsystemet.mdx

```diff
@@ -1,97 +1,94 @@
 
-I september 2023 hadde vi en åpen presentasjon der vi delte våre tanker rundt hvorfor vi trenger et felles designsystem. Opptaket fra presentasjonen kan du se under, og [her kan du se sammenstillingen av Miro-brettet](https://miro.com/app/board/uXjVMnk1ZM4=/?share_link_id=239809330169) der vi fikk over 440 tilbakemeldinger på gevinster og utfordringer knyttet til temaet.
+## Hva er et designsystem?
 
-<ResponsiveIframe
-  src='https://player.vimeo.com/video/862626520?h=bed84328b3&app_id=122963&title=0&byline=0&portrait=0&dnt=1'
-  width='100%'
-  height='600px'
-  frameBorder='0'
-  allow='autoplay; fullscreen; picture-in-picture'
-></ResponsiveIframe>
+Et designsystem består av UI-komponenter og mønstre som sikrer gjennomtenkte og konsistente brukeropplevelser i digitale tjenester.
 
-## Hva er felles designsystem?
+Ved å bruke grunnleggende byggeklosser fra designsystemet kan produktteamene selv bygge mer sammensatte komponenter som dekker spesifikke behov.
 
-Felles designsystem er en verktøykasse som skal bestå av grunnleggende designelementer, mønstre, god praksis og kodede komponenter som gir verdi å dele på tvers av offentlig sektor. Komponentene i designsystemet er allerede i bruk i [Altinn Studio](https://altinn.studio/) og flere offentlige etater har bidratt med komponenter de har hatt behov for i sin tjenesteutvikling.
+Et designsystem fungerer som en bro mellom design og utvikling, og sørger for at det er de samme verdiene vi jobber med.
 
-Komponentene i designsystemet skal være mulig å ta i bruk og utvide, uten å gjøre endringer i selve designsystemet. Ved å bruke prinsipper, tokens og komponenter fra designsystemet kan man selv bygge mer sammensatte komponenter som dekker spesifikke behov.
+## Hvorfor et _felles_ designsystem?
 
-## Hvorfor et felles designsystem?
+Regjeringen vil jobbe for at _"offentlige digitale tjenester skal ha et brukervennlig og gjenkjennbart design på tvers av ulike etater slik at innbyggerne med lav digital kompetanse kan benytte tjenesten"_. <sup>1</sup>
 
-Det er mange grunner til å ha et designsystem, den mest opplagte er nok at de som lager tjenester ikke trenger å finne opp hjulet på nytt hver gang de starter et nytt prosjekt. Grunnleggende komponenter kan plukkes fra et eksisterende designsystem, og på den måten får produktteamene mer tid å bruke på andre ting.
+Gjennom å samarbeide om et felles designsystem kan vi i større grad sikre helhetlige og gjenkjennelige brukeropplevelser på tvers av etater. I stedet for å lage de samme grunnkomponentene om og om igjen, kan vi lage dem skikkelig èn gang og sikre god kvalitet ved at at tilgjengelighetskrav og testing er godt ivaretatt. Best av alt - da har vi kun ett sted å vedlikeholde dem når ny teknologi, standarder, krav og behov oppstår. Det betyr mer tid til å løse viktigere problem for brukerne våre. 🎉
```

Comment:
Endringer uthevet
Gjennom å samarbeide om et felles designsystem kan vi i større grad sikre helhetlige og gjenkjennelige brukeropplevelser på tvers av etater. I stedet for å lage de samme grunnkomponentene om og om igjen, kan vi lage dem skikkelig **én** gang og sikre god kvalitet ved at at tilgjengelighetskrav og testing er godt ivaretatt. Best av alt - da har vi kun ett sted å vedlikeholde dem **når det kommer ny teknologi, standarder, krav og behov**. Det betyr mer tid til å løse viktigere problem for brukerne våre.

Reply:
Ble ikke uthevet ser jeg: Endret è til é og setnignen ...når det kommer